static_src/actions/domain_actions.js

```diff
@@ -0,0 +1,24 @@
+
+/*
+ * Actions for domain entities. Any actions such as fetching, creating,
+ * etc should go here.
+ */
+
+import AppDispatcher from '../dispatcher.js';
+import { domainActionTypes } from '../constants';
+
+export default {
+  fetch(domainGuid) {
+    AppDispatcher.handleViewAction({
+      type: domainActionTypes.DOMAIN_FETCH,
+      domainGuid
+    });
+  },
+
+  receivedDomain(domain) {
+    AppDispatcher.handleServerAction({
+      type: domainActionTypes.DOMAIN_RECEIVED,
+      domain
+    });
+  }
+};
```

Comment:
Ya, we can wait for the research team to tell us otherwise, but I have a feeling that the division is hard for most users to understand and that we'll actually be using them in concert.

I'm open either way, which direction do you want to go?